Frontend/ui-common/src/Components/PopUpModal/View.purs

```diff
@@ -133,6 +133,46 @@ view push state =
           , text $ state.secondaryText.text  
           , visibility $ state.secondaryText.visibility  
           ]
+          ,
+          linearLayout
```

Comment:
Create Seperate view function don't stack in single view function

Reply:
Done